scripts/composer/ScriptHandler.php

```diff
@@ -22,6 +22,15 @@ public static function createRequiredFiles(Event $event) {
     $drupalFinder = new DrupalFinderComposerRuntime();
     $drupalRoot = $drupalFinder->getDrupalRoot();
 
+    // If Drupal root was not found, exit.
```

Comment:
nit: follow the same code style as the rest of the file
```suggestion
    if (is_null($drupalRoot)) {
      $event->getIO()->writeError('<error>Drupal root could not be detected.</error>');
      exit(1);
    }
```

Reply:
Fair point! I followed more or less the calls from `checkComposerVersion`, but this looks much cleaner!